api/management/commands/loaddata.py

```diff
@@ -0,0 +1,47 @@
+import os
+import csv
+
+from django.core.management.base import BaseCommand
+
+from api.models import Disease, Sample
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
```

Comment:
Why is this function defined here? I've never seen argparse logic inside of a class. May just be that I'm unfamiliar with django workflows.